### PR TITLE
[doc] Fix commercial_mode_id for interregionalRail

### PR DESCRIPTION
**Pull request by prhod**
Looks like a typo when updating the specs

### CODE REVIEW

File: src/documentation/netex_idfm_to_ntfs.md [hunk 1]
```diff
@@ -175,7 +175,7 @@ metro | _any_ | Metro | Métro | Metro | Métro |
 rail | railShuttle | RailShuttle | Orlyval, CDG VAL | RailShuttle | Orlyval, CDG VAL |
 rail | suburbanRailway | LocalTrain | Train Transilien | LocalTrain | Train Transilien |
 rail | regionalRail | Train | TER / Intercités | regionalTrain | TER |
```
Comment (woshilapin) [inline src/documentation/netex_idfm_to_ntfs.md]: Seems like this is not the only typo...
```suggestion
rail | regionalRail | Train | TER / Intercités | regionalRail | TER |
```

**Comment by prhod** [comment]
thanks, i've made the modification

**Comment by woshilapin** [comment]
Oops, I think you've messed up the rebase or whatever you pushed on the branch :wink:

**Comment by prhod** [comment]
seems I forgot to rebase before pushing :p

**Pull request merged**